packages/dd-trace/src/lambda/handler.js

```diff
@@ -23,7 +23,15 @@ let __lambdaTimeout
  */
 function checkTimeout (context) {
   let remainingTimeInMillis = context.getRemainingTimeInMillis()
-  const apmFlushDeadline = parseInt(process.env.DD_APM_FLUSH_DEADLINE)
+
+  let apmFlushDeadline = process.env.DD_APM_FLUSH_DEADLINE_MILLISECONDS
+  // TODO: Remove this condition and parse int above
+  // once `datadog-lambda-js` updates tracer to v3
+  if (!apmFlushDeadline) {
```

Comment:
Update - we learned that the python change was not released yet so this env var can be removed w/o being backwards incompatible. Going to remove this conditional and tweak the logic a bit.

Reply:
updated logic and name